components/scream/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -332,16 +389,37 @@ class AtmosphereProcess : public ekat::enable_shared_from_this<AtmosphereProcess
     f.get_header_ptr()->get_tracking().add_customer(weak_from_this());
   }
 
-  virtual void set_required_field_impl (const Field<const Real>& f) = 0;
-  virtual void set_computed_field_impl (const Field<      Real>& f) = 0;
+  // The base class already registers the required and computed fields in
+  // the set_required/computed_field main routines.  These impl definitions
+  // provide a way for subclasses to define extra field bookkeeping that
+  // is not already covered by the `set_required/computed_field`.  It is
+  // not essential for subclasses to overwrite these two routines, but it
```

Comment:
Typo: overwrite->override. No need to change it here though (it would trigger another AT round, and require another approval).